config/config.go

```diff
@@ -217,7 +216,7 @@ type ScrapeConfig struct {
 	// Indicator whether the scraped metrics should remain unmodified.
 	HonorLabels bool `yaml:"honor_labels,omitempty"`
 	// A set of query parameters with which the target is scraped.
-	Params url.Values `yaml:"params,omitempty"`
+	Params map[string]string `yaml:"params,omitempty"`
```

Comment:
Relying on param order seems fishy to me. 3 is inconsistent with all other places where we use concatenation. Being able to use relabeling for many use cases rather than specific configs is great but we still have to keep an eye on usability, which mostly means consistency for relabeling.